cmd/minikube/cmd/start.go

```diff
@@ -820,8 +820,12 @@ func validateKubernetesVersions(old *cfg.Config) (string, bool) {
 
 	if nvs.LT(ovs) {
 		nv = version.VersionPrefix + ovs.String()
-		out.ErrT(out.Conflict, "Kubernetes downgrade is not supported, will continue to use {{.version}}", out.V{"version": nv})
-		return nv, isUpgrade
+		exit.WithCodeT(exit.Config, `Error: You have selected version {{.new}}, but your existing cluster for profile "{{.profile}}" is running version {{.old}}. Non-destructive downgrades are not supported, but you can proceed by performing one of the following options:
+
+* Run "minikube start [-p {{.profile}}] --kubernetes-version={{.old}}" or newer to continue using this cluster
```

Comment:
Remove the [] around the -p arguments here and elsewhere. You should expect that users will cut and paste this command everywhere.

If you wanted to make the output slightly smarter for the usual case where a user has not specified a custom profile name, you could do something like:

profileArg := ""
if  cfg.GetMachineName() != constants.DefaultMachineName {
  profileArg = fmt.Sprintf(" -p %s", cfg.GetMachineName())
}

and then just add profileArg to your exit message.